packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -48,6 +48,18 @@ describe('Cypress In Cypress', { viewportWidth: 1500 }, () => {
 
     cy.findByTestId('playground-num-elements').contains('3 Matches')
 
+    // This validates that each matching element is covered by the playground highlighting
+    cy.get('iframe.aut-iframe').its('0.contentDocument.body').then(cy.wrap).within(() => {
```

Comment:
This test was migrated from [packages/driver/cypress/e2e/e2e/selector_playground.cy.js](https://github.com/cypress-io/cypress/pull/20612/files#diff-f0d8ac2b2425c70eeb15b5b923110a83b8588c49aaba9bcadff90780a76afa1e) so that we could remove some test-only API [here](https://github.com/cypress-io/cypress/pull/20612/files#diff-2830a17a7fc1a0bbda7fd53965ecad693c7eae08406fe697ed6e84f07e0c2fa4L63)